Source/CoreClasses/User.swift

```diff
@@ -20,6 +20,7 @@ public class User {
   let userID: String
```

Comment:
Users now have an image attached. Seemed appropriate to do this as it used to transport the image from service call to being presented on the view